.github/workflows/loc-auto-merge.yml

```
@@ -1,5 +1,8 @@
 name: 'Auto-merge Validation'
 
+permissions:
```

also add a content permission actually or it's going to fail on the checkout 
```suggestion
permissions:
	contents: read
```

---

Sure, thanks!